src/main/java/uk/gov/hmcts/reform/pip/subscription/management/service/SubscriptionServiceImpl.java

```diff
@@ -192,14 +191,18 @@ private void duplicateSubscriptionHandler(Subscription subscription) {
      * @param subscriptionsList The list of subscriptions being sent
      */
     private void handleSubscriptionSending(UUID artefactId, List<Subscription> subscriptionsList) {
-        channelManagementService.getMappedEmails(subscriptionsList).forEach((email, listOfSubscriptions) -> {
-            SubscriptionsSummary summaryToSend =
-                formatSubscriptionsSummary(artefactId, email, listOfSubscriptions);
-
-            log.info("Summary being sent to publication services: " + summaryToSend);
-
-            publicationServicesService.postSubscriptionSummaries(summaryToSend);
-        });
+        List<Subscription> emailList = sortSubscriptionByChannel(subscriptionsList, Channel.EMAIL.notificationRoute);
+        List<Subscription> apiList = sortSubscriptionByChannel(subscriptionsList, "API");
```

Comment:
I think this should be Channel.API.notificationRoute

Reply:
updated
was going to do this however its not .API.notificationRoute, its API_COURTEL.notification route and didnt wanna confuse devs thinking they had to add API_OTHER.notificaiton route, i could just see duplicated code happening 😂 updated anyway and hoping for the best ;)